demo/capture_config.yaml

```diff
@@ -0,0 +1,140 @@
+lcr-modules:
+
+    _shared:
+        lcr-modules: "../"
+        lcr-scripts: "../../lcr-scripts/"
+        root_output_dir: "results/"
+        scratch_directory: "scratch/"
+        unmatched_normal_ids:
+            capture--grch37: "TCRBOA7-N-WEX"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           
+            grch37: " reference/genomes/grch37/downloads/hg19-grch37 "  
+        
+    mutect2:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+    
+    varscan:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+
+    liftover:
+        tool: "sequenza-1.4" 
+        dirs: 
+            _parent: "results/sequenza-1.4_liftover-1.2"
+        inputs:
+            sample_seg: "results/sequenza-1.4/03-igv_seg/genome--grch37/TCRBOA7-T-WGS--TCRBOA7-N-WGS--matched/filtered/sequenza_segments.igv.seg"
+
+    slms_3: 
+        inputs: 
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+
+    bam2fastq:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+        temp_outputs: True # fastq outputs will be temporary
+
+    manta:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+   
+    mixcr:
+        inputs:
+            sample_fastq_1: "data/{sample_id}.read1.fastq.gz"
+            sample_fastq_2: "data/{sample_id}.read2.fastq.gz"
+
+    sequenza:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+        scratch_subdirectories: []
+    
+    lofreq:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+        switches:
+            # Intentionally running LoFreq without a BED file for simplicity
+            # And to avoid having to include a large BED file in the repo
+            regions_bed:
+                _default: ""
+                capture: ""
+
+    gridss: 
+        inputs: 
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+        references: 
+            # See the current gridss module config file for details about where to obtain this file. 
+            viral_fa: "/projects/rmorin/projects/DLBCL_DHITsig_genomes/reference/gridss/refgenomes/human_virus/human_virus.fa"
```

Comment:
does this setup work when run on other systems? It is hard-coded to GSC path and needs to be updated, so the demo can run elsewhere. Didn't you face an error here when you first tried running it on Thanos? The same applies to the 2 paths in the next 2 lines.